tests/.nftignore

```diff
@@ -20,4 +21,6 @@ reports/fastqc/**/*_fastqc.{html,zip}
 reports/markduplicates/**/*.md.cram.metrics
 reports/snpeff/*/*_snpEff.csv
 reports/snpeff/*/snpEff_summary.html
+reports/vcftools/**/*.TsTv.qual
 variant_calling/**/*.vcf.{gz,gz.tbi}
+.DS_Store
```

Comment:
That should not be produce by Nextflow

Reply:
it ended up in my nf-test test files repeatedly